instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -372,11 +372,12 @@ def _get_system_cpu_time(self, observer: metrics.ValueObserver) -> None:
         """
         for cpu, times in enumerate(psutil.cpu_times(percpu=True)):
             for metric in self._config["system.cpu.time"]:
-                self._system_cpu_time_labels["state"] = metric
-                self._system_cpu_time_labels["cpu"] = cpu + 1
-                observer.observe(
-                    getattr(times, metric), self._system_cpu_time_labels
-                )
+                if hasattr(times, metric):
```

Comment:
How about getattr here followed by a check for `None`? Would save one attribute lookup if/when on hotpath.

Reply:
This would be a good optimization, I thought about it but then I remembered the default interval for this instrumentation was to collect metrics every 30 seconds. Can definitely be improved when it becomes an issue